Platform/UTMJailbreak.c

```diff
@@ -32,33 +36,62 @@ kern_return_t catch_exception_raise(mach_port_t exception_port,
                                     exception_type_t exception,
                                     exception_data_t code,
                                     mach_msg_type_number_t code_count) {
-    NSLog(@"Caught exception %d (this should be EXC_SOFTWARE), with code 0x%x (this should be EXC_SOFT_SIGNAL) and subcode %d. Forcing suicide.", exception, *code, code[1]);
+    fprintf(stderr, "Caught exception %d (this should be EXC_SOFTWARE), with code 0x%x (this should be EXC_SOFT_SIGNAL) and subcode %d. Forcing suicide.", exception, *code, code[1]);
     // _exit doesn't seem to work, but this does. ¯\_(ツ)_/¯
     return KERN_FAILURE;
 }
 
-void *exception_handler(void *argument) {
+static void *exception_handler(void *argument) {
     mach_port_t port = *(mach_port_t *)argument;
     mach_msg_server(exc_server, 2048, port, 0);
     return NULL;
 }
 
-static BOOL am_i_being_debugged() {
+static bool am_i_being_debugged() {
```

Comment:
It's probably better to [check `CS_DEBUGGED` using `csops`](https://github.com/hrydgard/ppsspp/blob/cf43f2d57f5ae850572374730f1c9fb38daba01d/ios/main.mm#L29), to be honest.